index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+  <html lang = "en" ng-app="Andejobs">
+    <head>
+      <title>Andejobs</title>
+      <link rel="stylesheet" href="libs/angular-material/angular-material.min.css">
+      <link rel="stylesheet" type="text/css" href="assets/app.css">
+      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=RobotoDraft:300,400,500,700,400italic">
+      <meta name= "viewport" content = "initial-scale=1"/>
+    </head>
+      <body> 
+        <div ng-controller="AndejobsCtrl">
+          <md-content>
+            <md-toolbar>
+              <div class="md-toolbar-tools md-accent">
+                <md-button class="md-icon-button" aria-label="Settings">
+                <!-- <md-icon md-svg-icon="/libs/angular-material/demos/toolbar/demoBasicUsage/img/icons/menu.svg"></md-icon> -->
+                </md-button>
+                  <h2>
+                    <span><md-button href="#/">Andela</md-button></span>
+                  </h2>
+                    <span flex></span>
+                    <md-button href="#/signin" ng-show="!token">Sign in</md-button>
+                    <md-button href="#/signup" ng-show="!token">Sign up</md-button>
+                    <md-button ng-click="signout()" ng-show="token">Sign Out</md-button>
+
+          </md-content>
+              </div>
+            </md-toolbar>
+         <div ng-view></div>
+        </div>
+        <script type="text/javascript" src="libs/angular/angular.js"></script>
+        <script type="text/javascript" src="libs/angular/angular.min.js"></script>
+        <script type="text/javascript" src="libs/angular-animate/angular-animate.min.js"></script>
+        <script type="text/javascript" src="libs/angular-material/angular-material.min.js"></script>
+        <script type="text/javascript" src="libs/angular-aria/angular-aria.min.js"></script>
+        <script type="text/javascript" src="libs/angular-route/angular-route.min.js"></script>
+        <script type="text/javascript" src="libs/ngstorage/ngStorage.min.js"></script>
```

Comment:
While you're developing your application, it's a good idea to use the uncompressed versions of the library.

Uncompressed versions are very useful during debugging. Minified versions (which you have included) are used when your code is to be pushed into production.